scripts/run-oneshot.sh

```diff
@@ -14,56 +11,25 @@ help()
   echo "- run the test"
   echo
   echo "Syntax: ./scripts/run-oneshot.sh [-a|d|f|h|t]"
-  echo "-a | --accelerator   The accelerator type, e.g. v2-8, v3-8, 8xv100, etc."
   echo "-d | --dryrun        Dryrun. If set, then the test does not run and only prints commands."
-  echo "-f | --file          The filepath for the test."
+  echo "-t | --test          The name of the test, e.g. `tf-nightly-classifier-resnet-func-v4-8-1vm`."
```

Comment:
It can be difficult to work out the exact test name, so it could be useful to point people to this script: https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/master/scripts/list-tests.sh